src/collection-view.js

```diff
@@ -295,7 +296,20 @@ Marionette.CollectionView = Marionette.AbstractView.extend({
       });
     }
 
-    return childView;
```

Comment:
Thanks for doing the research Paul!

I think multiple versions of BB on the page is definitely an edge case so, we should probably find a good work around.

I think it's a little bit of a regression if we have to tell everyone who has two versions to always do

```js
extend({
  childView: function() {return View}
})
```

Maybe a good compromise, would be to move the check to a global function that could be overriden?

```js
Marionette.isBackboneView 
```